src/components/pages/HomePage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate, useParams } from 'react-router-dom';
+import { NotFoundPage } from './NotFoundPage';
+
+export const HomePage = () => {
+  const { home } = useParams();
```

Comment:
I'm not quite sure why you're trying to destructure `home` from `useParams`. `useParams` is usually used to capture dynamic parts of the URL, and typically, the home page wouldn't have any dynamic parts. If you're trying to implement some sort of redirect logic, it might be better placed in a router or another component that handles routing.